css/styles.css

```diff
@@ -0,0 +1,42 @@
+
+.navbar-default{
+  background-color: transparent;
+  border-color: transparent;
+}
+.navbar-default .navbar-nav>.active>a, .navbar-default .navbar-nav>.active>a:focus, .navbar-default .navbar-nav>.active>a:hover{
+  color:#FFF;
+  background-color: rgba(17,17,17,.5);
+}
+.navbar-default .navbar-nav>li>a:focus, .navbar-default .navbar-nav>li>a:hover{
+  color:orange;
+}
+.navbar-default .navbar-nav>li>a{
+  color:#FFF;
+  text-shadow: 1px 1px 2px #0a0a0a;
+}
+
+.carousel-inner .item{
+  height: 80vh;
+  background-size: cover;
+  background-position: center;
+}
+.carousel-caption{
+  padding-bottom: 70px;
+}
+.first-slide{
+  background-image: url('../img/wood-2.jpg');
+}
+.slide-logo{
+  max-height: 300px;
+}
+.second-slide{
+  background-image: url('../img/Restaurant-2.jpg');
+}
+.third-slide{
+  background-image: url('../img/farm-2.jpg');
+}
+.btn-primary{
```

Comment:
change to: 
.btn-primary,
.btn-primary:hover{
  background-color: #F8512F;
  border-color: #F8512F;
}